addons/mail/static/src/models/messaging/messaging.js

```diff
@@ -369,7 +369,6 @@ registerModel({
             default: insertAndReplace(),
             isCausal: true,
             readonly: true,
-            required: true,
```

Comment:
why this change?

Reply:
I had actual crash in some tests of `messaging.time` being `undefined`

Reply:
I had this crash before `Chatter/thread` and fixing it, so I'll check again whether I still have the crash, and if that's still the case investigate what's going on

Reply:
Checked again and test crashes during teardown with `messaging.time` being undefined.

Apparently during teardown, we got to a point in which `messaging` exists but `messaging.time` no longer exists and there is no crash. `Chatter/_stopAttachmentsLoading` now triggers an update cycle due to props with attachment loading and timeout being turned into field. This is the actual update cycle that detects missing required on `messaging.time` and make the crash. Note that `Chatter/_stopAttachmentsLoading` is called in `Chatter/_willDelete`, so this is a nested update cycle.

Here I just removed the `required` on `Messaging/time` to allow same scenario as before with no crash with introduction of this update cycle that makes the check.

I tried keeping `required: true` on `Messaging/time` and not trigger update cycle from `Chatter/_stopAttachmentsLoading` (just kept `clearTimeout`), and tests pass successfully. I just pushed this version on this PR.

I think all of this shows that nested update cycles can be make checks on state, which is a problem because state of modelling makes sense before and after a full update cycle, not in-between.